src/compiler/performanceCore.ts

```diff
@@ -116,6 +98,4 @@ export function tryGetNativePerformanceHooks() {
  *
  * @internal
  */
-export const timestamp = nativePerformance ? () => nativePerformance.now() :
-    Date.now ? Date.now :
-    () => +(new Date());

```

Comment:
While here, removing this branch. Date.now is most definitely defined.

Reply:
It wasn't guaranteed to be when we targeted ES3, but since we've moved up our minimum target to run tsc this makes sense.